src/odemis/acq/test/fastem_test.py

```diff
@@ -589,6 +583,36 @@ def test_pre_calibrate(self):
                     continue
                 self.assertEqual(asm_config_orig[component][va], value)
 
+    def test_get_pos_first_tile(self):
+        """Test that the position of the first tile is calculated correctly for a varying number of fields."""
+        res_x, res_y = self.multibeam.resolution.value  # single field size
+        px_size_x, px_size_y = self.multibeam.pixelSize.value
+
+        # Loop over different ROA sizes by varying the number of fields in x and y.
+        for x_fields, y_fields in zip((1, 2, 40, 34, 5), (1, 22, 43, 104, 25)):
+            # The coordinates of the ROA in meters.
+            xmin, ymin, xmax, ymax = (0, 0, res_x * px_size_x * x_fields, res_y * px_size_y * y_fields)
+            coordinates = (xmin, ymin, xmax, ymax)  # in m
+
+            # Create an ROA with the coordinates of the field.
+            roa_name = time.strftime("test_megafield_id-%Y-%m-%d-%H-%M-%S")
+            roa = fastem.FastEMROA(roa_name, coordinates, None,
+                                   self.asm, self.multibeam, self.descanner,
+                                   self.mppc)
+
+            task = fastem.AcquisitionTask(self.scanner, self.multibeam, self.descanner,
+                                          self.mppc, self.stage, self.ccd,
+                                          self.beamshift, self.lens,
+                                          roa, path=None, future=None)
+
+            pos_first_tile_actual = task.get_pos_first_tile()
+
+            # The first tile is expected to be to the bottom right of the top left corner of the ROA in the
```

Comment:
This doesn't make sense to me. Why "bottom right".
With first tile you mean "The position of the first tile"?
Isn't the position of the first tile the center position of the top/left corner tile of the ROA?